### PR TITLE
Work preview

### PR DESCRIPTION
**Pull request by gestchild**
References: #4492

Implements new work preview designs (includes making page background white and restyling download):
Screenshot 2019-05-22 at 15 55 36

Screenshot 2019-05-22 at 15 39 30
Screenshot 2019-05-22 at 15 36 09
Screenshot 2019-05-22 at 15 37 00
Screenshot 2019-05-22 at 15 36 47

### CODE REVIEW

File: common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js [hunk 1]
```diff
@@ -277,14 +197,11 @@ const IIIFPresentationDisplay = ({
         previewThumbnails(
           iiifPresentationManifest,
           orderedStructuredImages(structuredImages(iiifPresentationManifest)),
-          4
+          5
```
Comment (davidpmccormick) [inline common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js]: is this supposed to be `ideaNumber`?

Comment (davidpmccormick) [reply common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js]: …I think you can ignore me – I was getting confused by the way GitHub collapsed the space from the function above.

File: common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js [hunk 2]
```diff
@@ -322,58 +239,33 @@ const IIIFPresentationDisplay = ({
...
-                  <span
-                    className={classNames({
-                      'flex-inline': true,
-                      'flex--v-center': true,
+            {imageThumbnails.map((pageType, i) => {
+              return pageType.images.map(image => {
+                return (
+                  <IIIFResponsiveImage
+                    key={image.id}
+                    lang={'en'}
```
Comment (davidpmccormick) [inline common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js]: is `lang` nullable? If so this would probably be better as `null`?

Comment (gestchild) [reply common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js]: yep, good spot, will change

**Pull request merged**